typed/merge.go

```diff
@@ -291,10 +301,10 @@ func (w *mergingWalker) indexListPathElements(t *schema.List, list value.List) (
 			continue
 		}
 		if _, found := observed.Get(pe); found {
-			errs = append(errs, errorf("duplicate entries for key %v", pe.String())...)
-			continue
+			observed.Insert(pe, value.NewValueInterface(nil))
```

Comment:
is it possible to *only* tolerate duplicates making the LHS and keep returning the duplicate error making the RHS?